packages/dbml-parse/src/lib/analyzer/validator/elementValidators/tableGroup.ts

```diff
@@ -70,11 +70,31 @@ export default class TableGroupValidator implements ElementValidator {
   }
 
   private validateSettingList(settingList?: ListExpressionNode): CompileError[] {
-    if (settingList) {
-      return [new CompileError(CompileErrorCode.UNEXPECTED_SETTINGS, 'A TableGroup shouldn\'t have a setting list', settingList)]
+    const aggReport = aggregateSettingList(settingList);
+    const errors = aggReport.getErrors();
+    const settingMap = aggReport.getValue();
+
+    for (const name in settingMap) {
+      const attrs = settingMap[name];
+      switch (name) {
+        case 'headercolor':
+          errors.push(...attrs.map((attr) => new CompileError(CompileErrorCode.INVALID_TABLE_SETTING, '\'headercolor\' is not supported', attr)))
+          break;
+        case 'note':
+          if (attrs.length > 1) {
+            errors.push(...attrs.map((attr) => new CompileError(CompileErrorCode.DUPLICATE_TABLE_SETTING, '\'note\' can only appear once', attr)))
+          }
+          attrs.forEach((attr) => {
+            if (!isExpressionAQuotedString(attr.value)) {
+              errors.push(new CompileError(CompileErrorCode.INVALID_TABLE_SETTING, '\'note\' must be a string literal', attr.value || attr.name!));
+            }
+          });
+          break;
+        default:
+          errors.push(...attrs.map((attr) => new CompileError(CompileErrorCode.INVALID_TABLE_SETTING, `Unknown \'${name}\' setting`, attr)))
```

Comment:
This only need the `default` and `note` case

Reply:
I added the `headercolor` case because Table has it, and we may support it for Table Group in the future